Features/DetailView/StackMovieDetailView.swift

```diff
@@ -170,7 +192,9 @@ class StackMovieDetailView: UIView {
         
         NSLayoutConstraint.activate([
             imageView.widthAnchor.constraint(equalTo: scrollContentStack.widthAnchor, multiplier: 0.61),
-            imageView.heightAnchor.constraint(equalTo: imageView.widthAnchor, multiplier: 1.41)
+            imageView.heightAnchor.constraint(equalTo: imageView.widthAnchor, multiplier: 1.41),
+            
+            reviewAndWatchlistStack.widthAnchor.constraint(equalTo: imageView.widthAnchor)
```

Comment:
I see. 
So, here I don't really set constraints to the width of the image since the width of the image itself is set according to the width of the stack. Did I get that right?

Reply:
When the alignment is set to `.fill` the cross axis (the opposite of the axis) will take the full size. So:
Horizontal stack with fill alignment: subviews will have the stack height as its height. 
Vertical stack with fill alignment: subviews will have the stack width as its widht.